### ISSUE TITLE
`tvctl on` distorts image in NTSC

### ISSUE THREAD

**Issue by nutki** [ali1234/raspi-teletext#37]
The change in 8d7bd58 causes `tvctl` to shift the even field display area up by 1 instead of the previous 2. Since the odd field is still shifted by 2, this causes a misalignment in the video display.
I have a [PR](https://github.com/ali1234/raspi-teletext/pull/36) that attempts to use the proper lines without modifying `tvctl`.
Maybe @ZXGuesser could verify this works as I don't have any device to decode CEA608 CC.

**Comment by ZXGuesser** [comment]
Yup that seems to be working, at least as far as the demo code repeating "hello!" is concerned with the datastream going in the first field.

**Comment by caiser01** [comment]
Two years late to this party but I've tested nutki's PR with several videos that have Line 21 data at the top of the frame. The video plays without any distortion or shifted fields and both Line 21 CC and TEXT data decode correctly on all of the TVs I've tried.

**Comment by csboling** [comment]
The patch is working for me to display captions on my LCD TV, and doesn't affect the display as the current main branch does. Interestingly I am only able to get captions on the digital display, my two CRT displays do not display captions when set to CC1 (with or without the patch).
